infrastructure/web/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "project" {
+  type    = string
+  default = "oak-national-academy"
+}
+
+variable "region" {
+  default = "europe-west2"
+}
+
+
+variable "retool_origin" {
+  type    = string
+  default = "https://creator.thenational.academy"
+}
+
+
+variable "environment" {
+  type    = string
+  default = "staging"
```

Comment:
I would keep all the defaults in step. It seems that the rest of the defaults are production defaults.